tests/core/actionrun_test.py

```diff
@@ -1932,7 +1928,7 @@ def test_recover_done_no_change(
         last_attempt.kubernetes_task_ic = "test-kubernetes-task-id"
 
         assert not mock_k8s_action_run.recover()
-        assert mock_cluster_repo.get_cluster.call_count == 0
+        assert mock_cluster_repo.get_cluster.call_count != 0
```

Comment:
not sure why this test just started failing in this branch, but this has been very wrong until now